de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/SarosSessionManager.java

```diff
@@ -491,15 +482,10 @@ public void invite(Collection<JID> jidsToInvite, String description) {
     for (JID jid : jidsToInvite) invite(jid, description);
   }
 
-  /**
-   * Adds project resources to an existing session.
-   *
-   * @param projectResourcesMapping
-   */
```

Comment:
why are the docs for this function deleted? did I miss something?

Reply:
that was not my intention... I fixed this in the new PR